### PR TITLE
use None

### PR DESCRIPTION
**Pull request by mfulton26**
`None` in Python is a very efficient placeholder for something not yet
defined

`is None` checks are also very fast (compare to `== None` or `== 0`)

### CODE REVIEW

File: Project1_2015.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+basement_position = None
 
 for index, char in enumerate(current_file):
     if char == '(':
         floor_counter += 1
     else:
         floor_counter -= 1
 
-    if floor_counter == -1 and basement_position == 0:
+    if basement_position is None and floor_counter == -1:
```
Comment (mfulton26) [inline Project1_2015.py:12]: checking `is None` first avoids `floor_counter ==  -1` check if `basement_position` is already not `None`

Comment (FormulaTranslator) [reply Project1_2015.py:12]: why is one check better than the other? I thought checking 1 variable is equal to checking another.

Comment (mfulton26) [reply Project1_2015.py:12]: http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html

Comment (FormulaTranslator) [reply Project1_2015.py:12]: never mind, I get it

**Review by FormulaTranslator — APPROVED** [review]

**Pull request merged**